bench/tx-generator/src/Cardano/Benchmarking/Script/Core.hs

```diff
@@ -632,6 +643,37 @@ createChangeGeneric submitMode createCoins addressMsg value count = do
   chunkList _ [] = []
   chunkList n xs = as : chunkList n bs where (as,bs) = splitAt n xs
 
+spendAutoScript :: SubmitMode -> FilePath -> ThreadName -> NumberOfTxs -> TPSRate -> ActionM ()
```

Comment:
It'd be nice to have a short verbal description of how this works.

Reply:
It does a binary search in the range of [0..10000] for the redeemer parameter of
the `loop.plutus` script ( lets call it n)  such that `loop 1 to n` is just within the resource limits. i.e. `loop 1 to (n+1)` is out of limits

Reply:
@MarcFontaine, sorry, I've meant in a code comment, since here the information will be lost.

Reply:
Yes there are hardly any comments in this code anywhere.
I think I have only added warnings and TODO comments to the really buggy parts of the code.